pyproject.toml

```
@@ -1,3 +1,9 @@
+[build-system]
+requires = [
+    "setuptools>=46.4.0",
```

Nitpick: it reads a bit better with spacing.
```suggestion
    "setuptools >= 46.4.0",
```